lib/api.rb

```diff
@@ -0,0 +1,28 @@
+require 'json'
+
+module ::Patreon
+  class Api
+
+    def self.campaign_data
+      get_json('/oauth2/api/current_user/campaigns?include=rewards,creator,goals,pledges&page[count]=200')
+    end
+
+    def self.get_json(uri)
+      response = get(uri)
+      return unless response.status == 200

```

Comment:
When the API request fails, we need to log an exception.

Sometimes the Patreon API fails and we get customer support requests and need this information to answer their questions.